sdk/eventhubs/azure-messaging-eventhubs/CMakeLists.txt

```diff
@@ -112,14 +117,10 @@ create_code_coverage(eventhubs azure-messaging-eventhubs azure-messaging-eventhu
 get_az_version("${CMAKE_CURRENT_SOURCE_DIR}/src/private/package_version.hpp")
 generate_documentation(azure-messaging-eventhubs ${AZ_LIBRARY_VERSION})
 
-if(BUILD_TESTING)
-  add_compile_definitions(TESTING_BUILD_AMQP)
-  if (NOT AZ_ALL_LIBRARIES OR FETCH_SOURCE_DEPS)
-    include(AddGoogleTest)
-    enable_testing ()
-  endif()
-  
-  add_subdirectory(test/ut)
+add_subdirectory(test)
```

Comment:
Keep this under `BUILD_TESTING`?

Reply:
I would, except it turns out that our convention puts both tests and perf tests in the same directory. And BUILD_TESTING is a separate flag from BUILD_PERFORMANCE.

So I guess I could have if(BUILD_PEFORMANCE || BUILD_TESTING) but that seems like it's going to fail as things move forward. Instead, we unconditionally include test and put the conditionals in the test directories's cmakelists.txt.

It's awkward but it's what we have. And I *really* don't want to have all the testing related goo in the main CMakeList.txt file, since it's a distraction from the tasks to build the component.